core/src/JsonLogic/Json.hs

```diff
@@ -148,6 +148,12 @@ parseFloat (JsonArray (a : _)) = parseFloat a
 -- Everything else is NaN
 parseFloat _ = notANumber
 
+parseFloat' :: Json -> Double
+-- Booleans are either 1 or 0.
+parseFloat' (JsonBool True) = 1
+parseFloat' (JsonBool False) = 0
+parseFloat' f = parseFloat f
+
```

Comment:
This is specific to comparison operations, so define it there. Perhaps rename it to `toNumeric`

Reply:
I think there are more functions that actually need this version, we need to look into which functions exactly, but I think it will not end up being specific to comparison. For example `{"-":[4,true]}` evaluates to 3.

Reply:
Ah, I found the common denominator:
https://tc39.es/ecma262/multipage/abstract-operations.html#sec-tonumber
So rename it to this instead and were good.